var/spack/repos/builtin/packages/cryptsetup/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Cryptsetup(AutotoolsPackage):
+    """Cryptsetup and LUKS - open-source disk encryption."""
+
+    homepage = "https://gitlab.com/cryptsetup/cryptsetup"
+    url      = "https://www.kernel.org/pub/linux/utils/cryptsetup/v2.2/cryptsetup-2.2.1.tar.xz"
```

Comment:
I would add:
```python
list_url = "https://www.kernel.org/pub/linux/utils/cryptsetup/"
list_depth = 1
```
This should allow `spack versions` and `spack checksum` to find new versions.